### PR TITLE
[WEB-1370] add docs for getQuoteV2

### PR DESCRIPTION
**Pull request by jerryafr**

### CODE REVIEW

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 1]
```diff
@@ -11,3 +11,304 @@ import { Callout } from "@/components";
...
+export interface QuoteResponseV2 {
+  srcChain: Chain;
+  srcAsset: Asset;
+  destChain: Chain;
+  destAsset: Asset;
+  amount: string;
+  quotes: (
+    QuoteDetails & {
+      boostQuote?: BoostedQuoteDetails // Only present if there is a boost opportunity for the requested swap amount
+      dcaParams?: { // Only present if the quote is a DCA quote. These parameters can be passed to the requestDepositAddress
```
Comment (zoheb391) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:148]:
```suggestion
      dcaParams?: { // Only present if the quote is a DCA quote. These parameters can be passed to requestDepositAddress
```

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 2]
```diff
@@ -11,3 +11,304 @@ import { Callout } from "@/components";
 # Get Quote V2
 
 ## getQuoteV2()
+
+Fetches the quotes for swapping tokens based on the provided `quoteRequest` and an `options` argument.
+Currently quotes can be either a normal quote or a DCA one.
```
Comment (zoheb391) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:16]: can you add a todo here to link to DCA docs

**Review by zoheb391 — APPROVED** [review]

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 3]
```diff
@@ -11,3 +11,304 @@ import { Callout } from "@/components";
...
+
+const quoteRequest = {
+  srcChain: Chains.Ethereum,
+  destChain: Chains.Bitcoin,
+  srcAsset: Assets.ETH,
+  destAsset: Assets.BTC,
+  amount: (1.5e18).toString(), // 1.5 ETH
+  brokerCommissionBps: 100, // 100 basis point = 1%
+  affiliateBrokers: [
+    { account: "cFM8kRvLBXagj6ZXvrt7wCM4jGmHvb5842jTtXXg3mRHjrvKy", commissionBps: 50 }
```
Comment (CumpsD) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:115]: What's the logic behind having to include affiliate broker accounts in a quote request?

I guess people can still just add up all commissionBps's and ask a quote with the total commission without having to specify every affiliate?

Comment (CumpsD) [reply pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:115]: (Unless the SDK is already doing this adding up, no idea what the HTTP endpoint looks like for v2)

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 4]
```diff
@@ -11,3 +11,304 @@ import { Callout } from "@/components";
 # Get Quote V2
 
 ## getQuoteV2()
+
+Fetches the quotes for swapping tokens based on the provided `quoteRequest` and an `options` argument.
+Currently quotes can be either a normal quote or a DCA one.
+The main difference from the old version (getQuote) is that the new version returns a list of quotes instead of a single one.
```
Comment (CumpsD) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:17]: Can you add some info on what this list can contain? Why is there a list of quotes and how to they differ from each other?

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx (hunk 3 above)
Comment (jerryafr) [reply pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:115]: that's the idea. SDK will add up the fees and use the sum for quoting. One can do this process themselves and just pass the whole brokerCommission here. but on creating the channel, they'd have to pass the values separately and correctly.
Having them separate in the quoting as well would make it easier for the implementers

Comment (CumpsD) [reply pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:116]: @jerryafr got it, thanks! I am using the HTTP Api, so I do this summing myself indeed, thanks

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx (hunk 4 above)
Comment (CumpsD) [reply pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:18]: Will you add more info on this in another PR?

**Pull request merged**